docs/grammar/BallerinaParser.g4

```diff
@@ -423,7 +422,15 @@ actionInvocationStatement
     ;
 
 transactionStatement
-    :   TRANSACTION LEFT_BRACE statement* RIGHT_BRACE transactionHandlers
+    :   TRANSACTION (WITH transactionPropertyInitExpressionList)? LEFT_BRACE statement* RIGHT_BRACE transactionHandlers
+    ;
+
+transactionPropertyInitExpression
+    : retriesStatement
+    ;
+
+transactionPropertyInitExpressionList
+    : transactionPropertyInitExpression (COMMA transactionPropertyInitExpression)*
```

Comment:
Should we allow a list here? By having a list I can have multiple retires expressions.
i.e:
```   
transaction with retries(4), retries(5), retries(6) {
 
} aborted {
 
}
```

Is this the expected behavior?